Simplenote/src/androidTest/java/com/automattic/simplenote/smoke/data/NoteDTO.java

```diff
@@ -0,0 +1,28 @@
+package com.automattic.simplenote.smoke.data;
+
+import java.util.List;
+
+public class NoteDTO {
```

Comment:
What does DTO stand for here?

Reply:
It is standing for the Data Transfer Object. The purpose is implementing a lightweight version of the model for tests.